docusaurus/docs/available-scripts.md

```diff
@@ -30,6 +30,8 @@ See the section about [deployment](deployment.md) for more information.
 
 If you aren’t satisfied with the build tool and configuration choices, you can `eject` at any time. This command will remove the single build dependency from your project.
 
-Instead, it will copy all the configuration files and the transitive dependencies (Webpack, Babel, ESLint, etc) right into your project so you have full control over them. All of the commands except `eject` will still work, but they will point to the copied scripts so you can tweak them. At this point you’re on your own.
+Instead, it will copy all the configuration files and the transitive dependencies (Webpack, Babel, ESLint, etc) right into your project as dependencies in `package.json`. You have full control over them and can always rearrange `package.json` as you deem reasonable. You can read [more here](https://stackoverflow.com/questions/44868453/create-react-app-install-devdepencies-in-dependencies-section)
```

Comment:
Maybe add a new paragraph after "...as dependencies in `package.json`" where we explain why they are installed as dependencies instead of dev dependencies.

Reply:
Alright

Reply:
@iansu Kindly review this one and let me know what you think